### DISCUSSION TITLE
More Frequent Data Pull from Inverter (the Power Flow in Real-Time feature)

### DISCUSSION THREAD

**Discussion by rmuser2** [hultenvp/solis-sensor#381] [Ideas]
Hi all,

Firstly, thank you for the work put into this. It has been a great help and automating my EV charging with respect to PV generation figures.

I note that we can only update the Solis app every 5 minutes, and Solis is no longer reducing this due to "server load". 

With that said, I note a feature in their app and web console behind a button called SET which allows for data to be updated as low as every 10 seconds.

Is there any scope for this to be included in Home Assistant? My guess is that it talks directly to the inverter rather than at the Solis servers.

Many thanks once again!

**Comment by jmason** [comment]
Interesting! I can't find this in my web console or app -- where do you see it appearing?

**Reply by cb4m** [reply]
Yes, interesting.  I have a 'SET' button at top right of power flow diagram on plant overview web page.  I doesn't seem to work properly yet though.  If I turn it on, the battery charge power seems to  get updated every 10s, but everything else (PV, Grid, Load) gets greyed out and shows zero.  I have an RAI-3K-48ES-5G with firmware updated in May.  The android app has a similar feature when clicking cog-wheel (from around v4.3.0?).    Hopefully it will be fully working soon!

**Comment by FrancYescO** [comment]
any news on this?
anyone checked if the "Upload Data Once Immediately" under Datalogger control allow to speedup the update process?
<img width="1361" height="280" alt="image" src="https://github.com/user-attachments/assets/b0b24be0-a3b5-46b6-8906-10b0c5ef7e6b" />
